core/src/main/java/com/lionparcel/commonandroid/form/LPTextArea.kt

```diff
@@ -0,0 +1,170 @@
+                errorTextEnabled = getBoolean(R.styleable.LPTextArea_errorTextEnabled, true)
+                counterTextEnabled = getBoolean(R.styleable.LPTextArea_counterTextEnabled, true)
+                supportedTextEnabled = getBoolean(R.styleable.LPTextArea_supportedTextEnabled, true)
+                labelText = getString(R.styleable.LPTextArea_textLabel).setString()
+                supportedText = getString(R.styleable.LPTextArea_textSupported).setString()
+                errorText = getString(R.styleable.LPTextArea_textError).setString()
+                edtTextEnabled = getBoolean(R.styleable.LPTextArea_enabledView, true)
+
+
+            }finally {
+                recycle()
+            }
+        }
+
+        clTextAreaParent = findViewById(R.id.clTextAreaParent)
+        txtAreaLabel = findViewById(R.id.txtAreaLabel)
+        llEditText = findViewById(R.id.llEditText)
+        lpEditTextArea = findViewById(R.id.lpEditTextArea)
+        txtAreaCounter = findViewById(R.id.txtAreaCounter)
+        txtAreaAlert = findViewById(R.id.txtAreaAlert)
+        txtAreaError = findViewById(R.id.txtAreaError)
+
+        // enable or disable LPTextArea
+        lpEditTextArea.isEnabled = edtTextEnabled
+        changeTextColorDisabled(edtTextEnabled)
+        // set LPTextArea label
+        txtAreaLabel.text = labelText
+        // set LPTextArea supported text
+        if (supportedTextEnabled) txtAreaAlert.text = supportedText  else txtAreaAlert.isVisible = false
+        // set LPTextArea error text
+        if (errorTextEnabled) txtAreaError.text = errorText else txtAreaError.isVisible = false
+
+        setTextInputEditText(counterMaxLength, hint, errorEnabled, errorTextEnabled, counterTextEnabled, supportedTextEnabled)
+
+    }
+
+    private fun setTextInputEditText(
+        maxLength : Int,
+        setHint : String,
+        isError: Boolean,
+        isErrorTextEnabled: Boolean,
+        isCounterTextEnabled : Boolean,
+        isSupportedTextEnabled : Boolean) {
+        lpEditTextArea.hint = setHint
+        txtAreaCounter.text = "0/$maxLength"
+        if (isError){
+            lpEditTextArea.onFocusChangeListener = OnFocusChangeListener{ view, b ->
+                if (b || !lpEditTextArea.text.isNullOrEmpty()){
+                    changeStateViewTextArea(!isError, isSupportedTextEnabled, isErrorTextEnabled)
+                } else if(!b && lpEditTextArea.text.isNullOrEmpty()) {
+                    changeStateViewTextArea(isError, isSupportedTextEnabled, isErrorTextEnabled)
+                }
+            }
+        }
+        if(isCounterTextEnabled){
+            lpEditTextArea.setMaxLength(maxLength)
+            lpEditTextArea.addTextChangedListener(object : TextWatcher {
+                override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+                }
+
+                override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
```

Comment:
if a method body is empty, it's ok to leave the argument name unchanged.
But if not empty, provide a more descriptive argument name.